Content Cards/src/content-card-5-blue.html

```diff
@@ -0,0 +1,13 @@
+
+   <div class='rounded-2xl w-72 text-black bg-white rounded-t-3xl rounded-b-3xl' >
+      <img class="rounded-t-2xl rounded-b-2xl" src="https://live.staticflickr.com/65535/52681290308_fc93d4ee7b_z.jpg" alt="img" />
+      <div class=' p-6'>
+        <h1 class=' text-gray-400 mb-2 font-bold tracking-[0.1px] '>10 June 2023</h1>
+        <h1 class='w-50  font-bold text-2xl text-blue-600 mt-2'> WaivlyFunds</h1>
+        <h1 class='font-normal mb-0 text-zinc-700 mt-2'>Slate helps you see how </h1>
+        <h1 class='font-normal mb-3 text-zinc-700'>many more days you need to work to reach. Slate helps you see how many
+          more days</h1>
+        <h1 class=' font-normal text-sm [text-decoration:underline] text-blue-600'>Know more</h1>
```

Comment:
make it an anchor tag with `href="/"` and `target="_blank"`.

Also use `underline` which is tailwind class, instead of `[text-decoration:underline]` @Tapo41